affiliates/philippines.swagger.json

```diff
@@ -0,0 +1,490 @@
+{
+    "openapi" : "3.0.0",
+    "info" : {
+      "title" : "Philippines Affiliate Documentation",
+      "description" : "# Introduction \n\nCredy is a company that specializes on the quality lead generation. \\\nThis document further describes how to integrate API to send leads to Credy.\\\nIf you have any questions, please do not hesitate to contact the IT manager at [it@credy.eu](mailto:it@credy.eu).\n\n## General information\n\nProduction URL: [https://api.credy.eu/v3/](https://api.credy.eu/v3/)\n\nStaging URL: [http://api.staging.credy.eu/v3/](http://api.staging.credy.eu/v3/)\n\nAPI supports post fields, XML and JSON.\n\nTo get response error messages in filipino, \"Accept-Language\" header’s value must be set to according ISO code. Ex: `fil` or `fil-PH`.\n## Authorization\nAuthorization is performed with OAuth 2.0 Client Credentials Grant.\nClient key and secret key will be assigned to you by Credy.\n## Lead registration flow\n\n1. Request is sent to /v3/customers\n\n2. If request is successful, customer uuid must be stored\n\n3. Request is sent to /v3/leads with previously stored customer uuid in query parameter \"customer\" *ex. /v3/leads?customer=a3ff7c20-e9e8-43a0-bfb7-71a5cf43b8d3* \n4. When the lead is accepted by api (under_investigation is received from POST lead), please redirect the customer to [https://www.credy.vn/loading/?lead={UUID}](https://www.credy.vn/loading/?lead={UUID}) (where {UUID} is replaced by the uuid that you receive from POST lead)\n",
+      "version" : "3.0.0"
+    },
+    "servers" : [ {
+      "url" : "http://api.staging.credy.eu/v3",
```

Comment:
if you want to specify servers, you can do that, but then you have to add dash at the end of server declaration, and remove slashes at the beginning of all paths

Reply:
adjusted